src/DrupalConsoleStack.php

```diff
@@ -0,0 +1,698 @@
+    }
+
+    /**
+     * Sets the password for the account with uid 1.
+     *
+     * @param string $accountPass
+     *   The password.
+     *
+     * @return $this
+     */
+    public function accountPass($accountPass)
+    {
+        $this->argForNextCommand('--account-pass=' . $accountPass);
+
+        return $this;
+    }
+
+    /**
+     * Returns the Drupal console version.
+     *
+     * @return string
+     *   The Drupal console version.
+     */
+    public function getVersion()
+    {
+        if (empty($this->drupalConsoleVersion)) {
+            $isPrinted = $this->isPrinted;
+            $this->isPrinted = false;
+            $result = $this->executeCommand($this->executable . ' --version');
+            $output = $result->getMessage();
+            $this->drupalConsoleVersion = 'unknown';
+            if (preg_match('#[0-9.]+#', $output, $matches)) {
+                $this->drupalConsoleVersion = $matches[0];
+            }
+            $this->isPrinted = $isPrinted;
+        }
+
+        return $this->drupalConsoleVersion;
+    }
+
+    /**
+     * Rebuilds the given cache.
+     *
+     * @param string $cacheName
+     *   The cache name.
+     *
+     * @return $this
+     */
+    public function cacheRebuild($cacheName = 'all')
+    {
+        $this->printTaskInfo('Cache rebuild');
+
+        return $this->drupal("cache:rebuild $cacheName");
+    }
+
+    /**
+     * Execute a specific Update N function in a module, or execute all.
+     *
+     * @param string $module
+     *   The module name.
+     * @param string $updateN
+     *   Specific update N function to be executed.
+     *
+     * @return $this
+     */
+    public function updateDb($module = 'all', $updateN = '')
+    {
+        $this->printTaskInfo('Perform database updates');
+        $this->drupal("update:execute $module $updateN");
+
+        return $this;
+    }
+
+    /**
+     * Sets the maintenance mode.
+     *
+     * @param bool $mode
+     *   Whether or not the maintenance mode should be on or off.
+     *
+     * @return $this
+     */
+    public function maintenance($mode = true)
+    {
+        $maintenanceMode = $mode ? 'on' : 'off';
+        $this->printTaskInfo("Set maintenance mode $maintenanceMode");
+
+        return $this->drupal('site:maintenance $maintenanceMode');
+    }
+
+    /**
+     * Execute a cron implementation.
+     *
+     * @param string $module
+     *   The module name.
+     *
+     * @return $this
+     */
+    public function executeCron($module)
+    {
+        $this->printTaskInfo('Execute cron');
+        $this->drupal("cron:execute $module");
+
+        return $this;
+    }
+
+    /**
+     * Executes `drupal site:install`.
+     *
+     * @param string $installationProfile
+     *   The installation profile to use during install.
+     *
+     * @return $this
+     */
+    public function siteInstall($installationProfile = '')
+    {
+        return $this->drupal('site:install ' . $installationProfile);
```

Comment:
We should be consistent, either use concatenation everywhere (my personal preference, but it seems the majority is the other way round in this file) or use double quotes with inline variables.